tests/ImportsNotSeparated/__snapshots__/ppsi.spec.ts.snap

```diff
@@ -164,7 +170,7 @@ import thirdParty from "third-party";
 import {
     random // inner comment
 } from './random';
-// leading comment
+// trailing comment for import-random (export statement is code that will be sorted to below imports, and so doesn't have precedence over the import-comment-attachment rules!)
```

Comment:
Is this a side-effect of how things are working, or a behavior that you think is correct?  

Personally, it seems to me like the only kind of trailing comments we should keep with an import are the ones that start on the same line as the end of the import declaration.  Anything else feels unexpected, since I never put a comment on a line after something that it's associated with.  But maybe some people do that!

Reply:
The situation here is that `export` is not considered part of the nodes we’re allowed to modify, but we do have logic to support pulling trailing comments around elsewhere, so the comment is preferentially bound.

I’ll tweak things to disable trailing comments except-on-same-line, and then you can review the result.

Reply:
This issue is gone with the latest commit! https://github.com/IanVS/prettier-plugin-sort-imports/pull/71/commits/fca03777f595716b1ae64753b9393de52dfbaede